ios/browser/api/ads/brave_ads.mm

```diff
@@ -1568,6 +1569,17 @@ - (bool)hasPrefPath:(const std::string&)path {
   return [self.prefs objectForKey:key] != nil;
 }
 
+- (void)setLocalStatePref:(const std::string&)path value:(base::Value)value {
+}
```

Comment:
Would be great to add `// Not needed on iOS`

Reply:
Subdivision targeting does work on iOS. However, the user is unable to change from Auto. Can you please confirm this will continue to work as before. Thanks